### PR TITLE
Clearer messaging for outputs

### LINKED ISSUE #927: Clearer messaging for outputs
All outputs are effectively optional. If a user specifies outputs, then the outputs will be written to the data model. If a user does not specify outputs then the outputs will not be written to the data model. In both cases, the user is performing the same analysis but the user will have to go into their Google Bucket to find the results if they don't specify outputs.

I think that we should include a clearer message if one or more outputs are blank, so the the user understands the consequences of not filling in all the outputs. Something like: 
>One or more outputs are blank. You can still run the analysis, but none of the outputs will be included in the data model. You will need to view the Google Bucket in order to find the outputs of the analysis.

This is related to sensible defaults in #871.

### PR DESCRIPTION
**Pull request by breilly2**
Fixes #927

### CODE REVIEW

File: src/pages/workspaces/workspace/tools/WorkflowView.js [hunk 1]
```diff
@@ -309,13 +315,14 @@ const WorkflowView = _.flow(
   render() {
     const { isFreshData, savedConfig, launching, activeTab, variableSelected, modifiedConfig } = this.state
     const { namespace, name, workspace } = this.props
+    const { rootEntityType } = modifiedConfig
```
Comment (elizzhao) [inline src/pages/workspaces/workspace/tools/WorkflowView.js]: I'm getting a "cannot read rootEntityType of undefined" error when I try to open the tools worfklowView on your branch. 
screen shot 2018-12-06 at 1 29 16 pm

Comment (breilly2) [reply src/pages/workspaces/workspace/tools/WorkflowView.js]: Thanks for looking. I rebased my branch before opening the PR and sorta rushed through a merge conflict. I did test it, though, and it seems to work for me. Is this a workspace you can share with me? (breilly.firecloud@gmail.com)

Comment (elizzhao) [reply src/pages/workspaces/workspace/tools/WorkflowView.js]: I'm using the badm test user on the son-of-jimmy workspace

Comment (breilly2) [reply src/pages/workspaces/workspace/tools/WorkflowView.js]: Should be fixed now. Something related to the initial render which is why I didn't catch it (even a reload works fine 🤷‍♂️ ). I prefer the updated code anyway.

**Review by zarsky-broad — APPROVED** [review]
My only question is if some of the labels should have punctuation at the ends of the strings.

**Comment by bradtaylor** [comment]
We thought about colons, but there's no punctuation in the mocks. I think the color differentiation and positioning make it pretty clear what elements the labels refer to.

**Pull request merged**